iocore/net/SSLCertLookup.cc

```diff
@@ -160,7 +160,40 @@ ticket_block_alloc(unsigned count)
 
   return ptr;
 }
+ssl_ticket_key_block *
+ssl_create_ticket_key_block_buffer(char *ticket_key_data, int ticket_key_len)
+{
+  ssl_ticket_key_block *keyblock = NULL;
+  int num_ticket_keys            = ticket_key_len / sizeof(ssl_ticket_key_t);
```

Comment:
As per the compiler warning ``num_ticket_keys`` and ``ticket_key_len`` should both be ``size_t``.